docs/getting-started/security-ui.asciidoc

```diff
@@ -16,210 +17,110 @@ image::images/search-bar.png[]
+Compare your Kubernetes infrastucture against a variety of security benchmarks. Refer to the <<findings-page>> to find out how to set this up.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
 [float]
-[[rules-page]]
-=== Rules page
+=== Timelines
 
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are interactive and allow you to share your findings with other team members. Refer to <<timelines-ui>> to learn more.
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
+
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
 
 [float]
-[[exceptions-page]]
-=== Exception lists page
+=== Cases
 
-The Exception lists page allows you to view and manage all rule exceptions. Refer to <<detections-ui-exceptions, Rule exceptions and value lists>> for more information about rule exceptions.
+Open and track security issues. Refer to <<cases-overview>> to learn more.
 
 [role="screenshot"]
-image::detections/images/exceptions-page.png[Shows the Exception lists page]
+image::cases/images/cases-home-page.png[Shows the Cases page]
 
 [float]
-[[hosts-ui]]
-=== Hosts page
+=== Explore
 
-The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
+Expand this section to view the following pages:
 
+* <<hosts-overview, *Hosts*>>: Examine key metrics for host-related security events using graphs, charts, and interactive data tables.
++
 [role="screenshot"]
 image::management/hosts/images/hosts-ov-pg.png[Shows the Hosts page]
-
-
-[float]
-[[network-page]]
-=== Network page
-
-The Network page provides key network activity metrics via an interactive map and network event tables
-that enable interaction with Timeline. Refer to <<network-page-overview, Network page overview>> for more information.
-
++
+* <<network-page-overview, *Network*>>: Explore the interactive map to discover key network activity metrics and investigate network events further in Timeline.
++
 [role="screenshot"]
 image::images/network-ui.png[Shows the Network page]
-
-[float]
-[[users-page-ov]]
-=== Users page
-
-The Users page provides a comprehensive overview of user data to help you understand authentication and user behavior within your environment. Refer to <<users-page, Users page>> for more information.
++
+* <<users-page, *Users*>>: Access a comprehensive overview of user data to help you understand authentication and user behavior within your environment.
```

Comment:
I think you need a `+` after this to move the following image under this bullet point.